scripts/create-new-package.bash

```diff
@@ -142,29 +144,128 @@ fi
 echo -e "${TERMINAL_COLOR_USER_NOTICE}The name '$MAINTAINER_NAME' and email address '$MAINTAINER_EMAIL' will be used as maintainer info!${TERMINAL_COLOR_NC}"
 
 
-# License
-echo -n -e "${TERMINAL_COLOR_USER_INPUT_DECISION}How do you want to licence your package? Current team license standard:['$TEAM_LICENSE']: ${TERMINAL_COLOR_NC}"
-read LICENSE
-LICENSE=${LICENSE:=$TEAM_LICENSE}
+# License options for a multiple choice
+license_user_input_option="user input"
+licence_team_option="Current team license standard: ['$TEAM_LICENSE']"
+
+license_apache_option="Apache-2.0"
+license_apache_desc="A permissive license with an added patent grant."
```

Comment:
Where did you get those licenses and description? Can we use the source directly before we hard-code everything into our code?